examples/shared/storybook.js

```diff
@@ -0,0 +1,42 @@
+const selectors = {
+  toolbarButtonByTitle: function (title) {
+    return `button[title="${title}"]`;
+  },
+  tabByTitle: function (title) {
+    return `button[role="tab"]:text("${title}")`;
+  },
+  storyByPath: function (path) {
+    const parts = path.toLowerCase().split("/");
+
+    if (path[0] === "/") parts.shift();
+
+    const story = parts.pop();
+    return this.storyByItemId(
+      story ? `${parts.join("-")}--${story}` : parts.join("-")
+    );
+  },
+  storyByItemId: function (id) {
+    return `[data-item-id="${id}"]`;
+  },
+};
+
+// Navigates to either a story or a folder (if the path includes a trailing slash).
+// A leading slash is optional.
+const openStory = async (action, path, callback) => {
+  await action(`Open story: ${path}`, async (page, ...rest) => {
+    // await Promise.all([
+    // page.waitForNavigation(),
```

Comment:
I assume we dont need this waitForNavigation

Reply:
Nope. I cleaned this up locally but missed it in my commit. `waitForNavigation` seems to only be required for full-page reloads (and then maybe only when it's not triggered by click/form submit). Cleaned it up!